### PR TITLE
Tiger blocks simplification by state

### LINKED ISSUE #512: Tiger blocks simplification by state
The Tiger blocks layer is huge and we can't simplify it using mapshaper because of OOMs. We are simplifying it using the PostGIS built-in functions (`ST_SimplifyVW`) but we are losing the topology that mapshaper ensures:

simplification-do

To avoid the problem and achieve a layer with some topology we need to divide and simplify the layer by state using mapshaper.

### PR DESCRIPTION
**Pull request by antoniocarlon**
Simplifying the Tiger blocks by state to take advantage of the mapshaper topologically correct simplification
Closes #512

### CODE REVIEW

File: tasks/us/census/tiger.py [hunk 1]
```diff
@@ -385,6 +388,95 @@ def output(self):
...
+        return simplifications
+
+    def run(self):
+        session = current_session()
+        for _, table in self.input().items():
+            query = '''
+                    CREATE TABLE IF NOT EXISTS {output} AS
+                    SELECT *
+                    FROM "{schema_input}".{table_input}
+                    WHERE 1=0
```
Comment (ethervoid) [inline tasks/us/census/tiger.py]: Are you only creating the table without rows?

File: tasks/us/census/tiger.py [hunk 2]
```diff
@@ -385,6 +388,95 @@ def output(self):
...
+                    '''.format(schema_input=table.schema,
+                               table_input=table.tablename,
+                               output=self.output().table)
+            session.execute(query)
+
+            query = '''
+                    INSERT INTO {output}
+                    SELECT *
+                    FROM "{schema_input}".{table_input}
```
Comment (ethervoid) [inline tasks/us/census/tiger.py]: Why do you make it in two steps?

Comment (antoniocarlon) [reply tasks/us/census/tiger.py]: These `create` + `insert` are inside a `for` loop that iterates over all the simplified tables. This inserts into the output table every iteration but creates the table only on the first one (`IF NOT EXISTS`)

Comment (ethervoid) [reply tasks/us/census/tiger.py]: offline -> `input_table` has all the same columns. First create table creates a table with the output table not with the input table and then start with input

**Pull request merged**